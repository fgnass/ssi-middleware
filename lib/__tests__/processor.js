const processor = require('../processor');

const ssi = processor({
  file: path => `file:${path}`,
  virtual: path => `virtual:${path}`
});

function spec(name, html) {
  test(name, () => {
    return ssi(html).then(res => expect(res).toMatchSnapshot());
  });
}

spec('file', '<html><!--#include file="/path"--></html>');
spec('virtual', '<html><!--#include virtual="/path"--></html>');
