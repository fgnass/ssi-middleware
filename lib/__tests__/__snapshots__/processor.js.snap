// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file 1`] = `"<html>file:/path</html>"`;

exports[`virtual 1`] = `"<html>virtual:/path</html>"`;
